target-list-with-crowdsec/update-target-list.sh

```diff
@@ -17,7 +17,7 @@ json_payload+=$(echo "$banned_ips" | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/","/g
 json_payload+=']
 }'
 
-http_code=$(curl -s -w %{http_code} --request PATCH \
+http_code=$(curl -s -w %{http_code} -o /dev/null --request PATCH \
```

Comment:
I assume the `<your-firewalla-domain>` part will be replaced somewhere before this script can be executed, right ?